crates/egui/src/containers/mod.rs

```diff
@@ -21,7 +21,7 @@ pub use {
     combo_box::*,
     frame::Frame,
     modal::{Modal, ModalResponse},
-    panel::{CentralPanel, SidePanel, TopBottomPanel},
```

Comment:
We should keep the old `SidePanel` and `TopBottomPanel` as deprecated aliases for `Panel`, to help out users migrate